serverless.yml

```diff
@@ -2,13 +2,22 @@ service: scale-of-belief-import
 
 provider:
   name: aws
-  runtime: nodejs8.10
-  stage: dev
+  stage: ${env:ENVIRONMENT}
+  runtime: nodejs10.x
   region: us-east-1
-  versionFunctions: false
+  vpc:
+    securityGroupIds:
+      - ${file(${env:ECS_CONFIG}/bin/vars.yml):us-east-1.webapp_elb_sg}
+    subnetIds: ${file(${env:ECS_CONFIG}/bin/vars.yml):us-east-1.prod_apps_1_all}
+  stackTags:
+    Name: ${env:PROJECT_NAME}-${self:custom.environmentMap.${env:ENVIRONMENT}}
+    owner: apps@cru.org
+    application: ${env:PROJECT_NAME}
+    env: ${self:custom.environmentMap.${env:ENVIRONMENT}}
+    managed_by: serverless-framework
+    function: lambda
   memorySize: 256
   environment: ${file(serverless/environment.js)}
-  $<<: ${file(serverless/provider.js)}
```

Comment:
I'm not understanding how the `provider.js` functionality is migrating over. Guessing it has to do with webpack, but not sure.

Reply:
I'm removing the `sls-production.yml` provider files from ecs_config. Only 1 project used it and it made things more complicated.

Reply:
Everything in the sls file was added directly to the project.
https://github.com/CruGlobal/scale-of-belief-import/pull/20/files/d1528aadf4a577be676a1ad6a5b5daaee2c0f1e2#diff-220045f50daee3e35e21b9d5fe19d67eR8-R18